inst/extdata/OSD/B/BERN.json

```diff
@@ -75,12 +75,12 @@
         "name": "Ap1",
         "top": 0,
         "bottom": 8,
-        "dry_hue": "NA",
-        "dry_value": "NA",
-        "dry_chroma": "NA",
-        "moist_hue": "10YR",
-        "moist_value": 5,
-        "moist_chroma": 3,
+        "dry_hue": "10YR",
+        "dry_value": 5,
+        "dry_chroma": 3,
+        "moist_hue": "NA",
```

Comment:
BERN gives broken and crushed moist colors that are not parsed: `"Ap2--3 to 9 inches; brown (10YR 5/3) silt loam, dark brown (7.5YR 3/2) broken and crushed moist; "`

Reply:
Tagged this case in an issue to add better support for multiple colors / sample preparations: https://github.com/ncss-tech/SoilKnowledgeBase/issues/82